.github/workflows/algolia-upload-index.yml

```
@@ -22,4 +22,4 @@ jobs:
       - run: yarn install
 
       - name: Sync Algolia Index
-        run: yarn exe scripts/sync-algolia-index.ts
+        run: yarn exe site/scripts/sync-algolia-index.ts
```

```suggestion
        run: yarn workspace @blockprotocol/site scripts/sync-algolia-index.ts
```

(both variants are generally equal, the main difference is `process.cwd()`